### DISCUSSION TITLE
Can etcd watcher read older events?

### DISCUSSION THREAD

**Discussion by arjunsingriuber** [etcd-io/etcd#18042] [Q&A]
Can etcd 3.5 watcher read older events? If yes, how? I inserted a few thousand key/values in etcd 3.5 and brought the revision to 75,000. But when I start the watcher to watch from revision 1000, I don't get any events.

**Comment by ahrtr** [comment]
> But when I start the watcher to watch from revision 1000, I don't get any events.

Most likely the key you were watching doesn't exist at all if you did not see any error. If it isn't the case, please provide detailed steps.

**Reply by arjunsingriuber** [reply]
Resolved here - https://github.com/etcd-io/etcd/issues/18053#issuecomment-2124631971
